CHANGELOG.md

```
@@ -2,44 +1,31 @@
-
 ## [2.3.1](https://github.com/VilnaCRM-Org/user-service/compare/v2.3.0...v2.3.1) (2025-02-04)
 
+## Bug Fixes
 
-### Bug Fixes
-
-* **#137:** load test reports and error fix ([#138](https://github.com/VilnaCRM-Org/user-service/issues/138)) ([87b9f48](https://github.com/VilnaCRM-Org/user-service/commit/87b9f48b482bb824e7ea8d9592b34a02b04d66e7)), closes [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137)
-
-
-
+* **#137:** load test reports and error fix ([#138](https://github.com/VilnaCRM-Org/user-service/issues/138)) ([87b9f48](https://github.com/VilnaCRM-Org/user-service/commit/87b9f48b482bb824e7ea8d9592b34a02b04d66e7)), closes [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137) [#137](https://github.com/VilnaCRM-Org/user-service/issues/137)
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)